tests/Application/src/Grid/AuthorWithBooksWithUseOutputWalkersEnabled.php

```diff
@@ -38,7 +38,8 @@ public function buildGrid(GridBuilderInterface $gridBuilder): void
             ->setRepositoryMethod(["expr:service('app.authors_with_books_query_builder')", 'create'])
             ->addField(
                 StringField::create('book')
-                ->setSortable(true, 'book.title'),
+                    ->setPath('books[0].title')
+                    ->setSortable(true, 'book.title'),
```

Comment:
> ->setPath('books[0].title')

Feels weird, but I don't have any points.

Reply:
yeah, not very cool but enough for the test itself.